ansible/roles/vttablet/templates/vttablet@.service.j2

```diff
@@ -22,12 +22,19 @@ LimitMEMLOCK=infinity
 WorkingDirectory={{vitess_root}}
 User={{vitess_user}}
 ExecStart=/bin/bash -c 'vttablet \
+	{% if vitess_version_name == "latest" or vitess_version_name == "v_15" %}
+		--db_app_user=vt_app \
+		--db_dba_user=vt_dba \
+		--db_filtered_user=vt_filtered \
+		--db_repl_user=vt_repl \
+	{% else %}
+		--db-config-app-uname=vt_app \
+		--db-config-dba-uname=vt_dba \
+		--db-config-filtered-uname=vt_filtered \
+		--db-config-repl-uname=vt_repl \
+	{% endif %}
```

Comment:
You don't need the if, else. The parameters `db_app_user` are very old. You can just use them.

Reply:
gotcha

Reply:
done via e8d448a